compose/completion.md

```diff
@@ -11,54 +11,72 @@ for the bash and zsh shell.
 
 ### Bash
 
-Make sure bash completion is installed. If you use a current Linux in a non-minimal installation, bash completion should be available.
-On a Mac, install with `brew install bash-completion`
+Make sure bash completion is installed. If you use a current Linux in a
+non-minimal installation, bash completion should be available. On a Mac, install
+with `brew install bash-completion`.
 
-Place the completion script in `/etc/bash_completion.d/` (`/usr/local/etc/bash_completion.d/` on a Mac), using e.g.
+Place the completion script in `/etc/bash_completion.d/`
+(`/usr/local/etc/bash_completion.d/` on a Mac):
 
-    curl -L https://raw.githubusercontent.com/docker/compose/$(docker-compose version --short)/contrib/completion/bash/docker-compose -o /etc/bash_completion.d/docker-compose
+```shell
+curl -L https://raw.githubusercontent.com/docker/compose/master/contrib/completion/bash/docker-compose -o /etc/bash_completion.d/docker-compose
```

Comment:
You will need sudo on this. Is it possible to put it somewhere local to the user, like the ZSH example below, instead?